corehq/apps/userreports/tests/test_report_aggregation.py

```diff
@@ -64,16 +66,69 @@ def _create_data_source(cls):
                 },
             ],
         )
+        # this is a hack to have both sql and es backends created in a class
+        # method. alternative would be to have these created on each test run
```

Comment:
didn't look with a fine tooth comb, but is the whole thing copy/pasted except the backend ID? seems like it would be more DRY to make these in a loop over `backend_id`s